yarn.lock

```diff
@@ -26298,6 +26298,11 @@ mock-fs@5.1.1:
   resolved "https://registry.yarnpkg.com/mock-fs/-/mock-fs-5.1.1.tgz#d4c95e916abf400664197079d7e399d133bb6048"
   integrity sha512-p/8oZ3qvfKGPw+4wdVCyjDxa6wn2tP0TCf3WXC1UyUBAevezPn1TtOoxtMYVbZu/S/iExg+Ghed1busItj2CEw==
 
+mock-fs@5.1.2:
+  version "5.1.2"
+  resolved "https://registry.npmjs.org/mock-fs/-/mock-fs-5.1.2.tgz#6fa486e06d00f8793a8d2228de980eff93ce6db7"
```

Comment:
Looks like a PR (prob the external one against `npm/vue` had `npm install` run instead of `yarn` - should we re-generate the `yarn.lock` as part of this to point at `registry.yarn` instead of `npmjs`?

Reply:
sounds like we should. Is the easiest place to do this inside the `npm/vue` package or delete the `yarn.lock` in `npm/vue` and run `yarn` from the root directory?

Reply:
I just replaced it manually in the `yarn.lock` and it seems fine: https://github.com/cypress-io/cypress/pull/21416/commits/796ac005ed15f6b9a25f0f05cc1080aa942d1800

Non-standard but I think since we are just changing the endpoint, not versions, it's fine.